src/permission-set.js

```diff
@@ -736,7 +738,10 @@ class PermissionSet {
     if (!webClient) {
       return Promise.reject(new Error('Cannot save - no web client'))
     }
-    return webClient.put(aclUrl, this.serialize(rdf))
+    contentType = contentType || DEFAULT_CONTENT_TYPE
+    let graph = this.serialize(rdf, contentType)
+    return Promise.resolve()
```

Comment:
this.serialize() returns a Promise, so currently it looks that a Promise gets passed to webClient.put as graph